### PR TITLE
feat(python): fastapi server codegen now supports file upload and download

### PR DESCRIPTION
**Pull request by armandobelardo**
Fixes FER-2782

The title says it all, this still needs to be tested, but the codegen looks sane

This supports upload and download, for upload I inlined all the parameters, since I wasn't seeing in FastAPI docs if you could have file upload params within a model, and it felt more sane to do this then to split up the object into inlining file upload properties and referencing body properties as an object

### CODE REVIEW

**Comment by dsinghvi** [comment]
@armandobelardo codegen looks good to me too -- will hold off on approving till after you test

File: seed/fastapi/plain-text/resources/service/service/service.py
Comment (dsinghvi) [inline seed/fastapi/plain-text/resources/service/service/service.py:70]: know why this changed?

**Review by dsinghvi — APPROVED** [review]

File: seed/fastapi/plain-text/resources/service/service/service.py
Comment (armandobelardo) [reply seed/fastapi/plain-text/resources/service/service/service.py:70]: Yeah will update, this is technically correct, but will revert for strings to maintain parity

**Pull request merged**
